src/components/Pool/SortByDropdown/SortByDropdown.component.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useContext, useRef } from 'react'
+import { ThemeContext } from 'styled-components'
+import Popover from '../../Popover'
+import { useTranslation } from 'react-i18next'
+import { Text, Flex } from 'rebass/styled-components'
+import { ChevronDown } from 'react-feather'
+import { useOnClickOutside } from '../../../hooks/useOnClickOutside'
+import { LIQUIDITY_SORTING_TYPES } from '../../../constants'
+import { List, ListItem, StyledFlex, StyledText } from './SortByDropdown.styles'
+
+export function SortByDropdown({
+  sortBy,
+  onSortByChange,
+}: {
+  sortBy: string
+  onSortByChange: (sortBy: string) => void
+}) {
+  const popoverRef = useRef()
+  const { t } = useTranslation()
+  const [isOpen, setIsOpen] = useState(false)
+  const theme = useContext(ThemeContext)
+  useOnClickOutside(popoverRef, () => setIsOpen(false))
+
+  const sortOptions = {
```

Comment:
I'm wondering if we need to translate it ? Maybe for now we can just use `enum` ? Also this variable will be recreated on each re-render

Reply:
Will use `enum` for now and add translations in a next step.